src/main/java/graphql/nadel/DefinitionRegistry.java

```diff
@@ -42,17 +44,30 @@ public SchemaDefinition getSchemaDefinition() {
         return (SchemaDefinition) definitionsByClass.get(SchemaDefinition.class).get(0);
     }
 
+    public Map<OperationType, ObjectTypeDefinition> getOperationTypes(){
+        Map<OperationType, ObjectTypeDefinition > opsTypesMap = new HashMap<>();
+
+        Arrays.stream(OperationType.values()).forEach(opsType ->
+                opsTypesMap.put(opsType, getOpsType(opsType.getOpsType(), opsType.getDisplayName())));
+
+       return opsTypesMap;
```

Comment:
Personally I think for simple`for` loops are better than streams.  I guess its a bit personal choice but I think it reads nicer

```
for (opsType : OperationType.values()) {
  opsTypesMap.put(opsType, getOpsType(opsType.getOpsType(), opsType.getDisplayName());
}
```

Reply:
Or use stream and Collectors.toMap